src/cfml/system/modules_app/forgebox-commands/commands/forgebox/show.cfc

```diff
@@ -130,17 +130,18 @@ component aliases="show" {
 				print.line();
 				print.blackOnWhite( ' #entryData.title# ' )
 					.boldText( '   ( #entryData.user.fname# #entryData.user.lname#, #entryData.user.username# )' )
-					.boldGreenLine( '   Rating: #repeatString( '*', val( entryData.avgRating ) )#' );
+					.boldGreen( '   Rating: #repeatString( '*', val( entryData.avgRating ) )#   ' )
+					.boldWhiteOnRedLine( entryData.listed ? "" : "  Unlisted  " );
```

Comment:
It technically has two extra spaces, but I didn't think it was a big deal.  If you'd like, I can clean it up, but the code will be a bit more messy.

Reply:
Ok, cool.  If you've looked at it with in the CLI and it looks fine then I'm ok with it.

Reply:
Can you add a quick elvis operator on that.  If I merge this right now, it will break anyone who updates to the bleeding edge prior to us pushing out the Forgebox.io piece.